src/geoserver/util.py

```diff
@@ -50,6 +52,20 @@
 DEFAULT_DB_STORE_NAME = 'postgresql'
 
 
+def error_handling_decorator(func):
+    def inner(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except HTTPError as exc:
+            logger.warning(f'traceback={traceback.format_exc()},\nresponse={exc.response.text}')
+            raise exc
```

Comment:
Create new LaymanError in error_list.py: `52: (400, 'GeoServer HTTP or connection error.')`


`raise exc` -> `raise LaymanError(52, data={'caused_by': exc.__class__.__name__, 'http_code': exc.response.status_code}, private_data={'response_text': exc.response.text}) from exc`